linodecli/configuration/__init__.py

```diff
@@ -1,19 +1,17 @@
 """
-Handles configuring the cli, as well as loading configs so that they can be
-used elsewhere.
+Configuration helper package for the Linode CLI.
 """
 
-import argparse
-import os
-import sys
-from typing import Dict
-
+# Private methods need to be imported explicitly
+from .auth import *
 from .auth import (
     _check_full_access,
     _do_get_request,
     _get_token_terminal,
     _get_token_web,
 )
+from .config import *
+from .helpers import *
 from .helpers import (
```

Comment:
Exporting these private methods isn't ideal but it's necessary to maintain compatibility with certain other components of the CLI without making large changes

Reply:
If they're necessary can we just make public wrappers or convert them to public?